### ISSUE TITLE
Impaired first-phase insulin secretion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11674]
# Impaired first-phase insulin secretion

## Proposed definition

Reduction or delay of the early, rapid phase of insulin release that normally occurs within approximately ten minutes following a glucose load.

## Parent term

- `HP:0006279 Beta-cell dysfunction`

## Synonyms

- Reduced acute-phase insulin response
- Impaired early-phase insulin release

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001516
- label: Impaired first-phase insulin secretion
- parents: [Abnormal glucose homeostasis (HP:0011014)](https://hpo.jax.org/app/browse/term/HP:0011014)
- pmid: PMID:15308473|PMID:11424229
- def: An abnormal reduction in the ability to produce first-phase insulin secretion, which is defined as the initial burst of insulin released in the first 5 to 10 minutes after the beta cell is exposed to a rapid increase in glucose (or other secretagogues).
- comment: In patients with impaired glucose tolerance or in the early stages of type 2 diabetes, first-phase insulin release is almost invariably lost despite the enhancement of second-phase secretion.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11674
- synonyms:
